docs/installation/devs.rst

```diff
@@ -156,26 +160,30 @@ packages with pip. In the directory where your copy of mrsimulator is, type:
     $ pip install -r requirements-dev.txt
     $ pip install -e .
 
-As always, if you get an error that you don’t have the permission to install the
+As before, if you get an error that you don’t have the permission to install the
 package into the default site-packages directory, you may try installing by adding the
 ``--user`` option.
 
 
-Note for the developers and contributors
-''''''''''''''''''''''''''''''''''''''''
+Note for the developers and contributors.
+'''''''''''''''''''''''''''''''''''''''''
 
-**Before commits**: Mrsimulator follows python community standards for writing code and
-documentation. To help guide the developers and contributors towards these standards,
-we have created a *.pre-commit-config.yaml* file, that when used with ``pre-commit``, will
-inspect the code and document for issues.
-Type ``pre-commit run`` before git commits to inspect the changes.
+**Before commits**: Mrsimulator follows Python community standards for writing code and documentation.
+To help guide the developers and contributors toward these standards, we have created
+a *.pre-commit-config.yaml* file that, when used with ``pre-commit``, will inspect
+the code and document for issues. At the root directory of the mrsimulator package folder, type
```

Comment:
Do we want to use package when referencing the mrsimulator directory? The word package is suggestive of code which performs some function, but is otherwise hidden to the user. Here mrsimulator is a real folder (or a real as data on a computer can be).

Reply:
Not sure.  @deepanshs ?

Reply:
Yeah, remove the package.